app/models/assignment.rb

```diff
@@ -8,6 +8,13 @@ class Assignment < ActiveRecord::Base
   belongs_to :assigned_by_user, class_name: "User"
   belongs_to :target, polymorphic: true
 
+  enum priority: {
```

Comment:
I'm not familiar with `enum` but does it add a validation to ensure that the value being written into the DB is within the list below? Kind of related to my comment in https://github.com/discourse/discourse-assign/pull/317/files#r853892117.

Reply:
(see above comment)